applications/launchpad/gui-react/src/components/Callout/index.tsx

```diff
@@ -0,0 +1,38 @@
+import Text from '../Text'
+import { CalloutIcon, StyledCallout } from './styles'
+
+import { CalloutProps } from './types'
+
+/**
+ * Callout component that renders styled box with proper icon.
+ * NOTE: It supports only the `warning` type for now.
+ *
+ * @param {CalloutType} [type='warning'] - the callout type/style.
+ * @param {ReactNode} [icon] - override the icon.
+ * @param {string | ReactNode} children - the callout content (text or ReactNode).
+ */
+const Callout = ({
+  type = 'warning',
+  icon = '⚠️',
```

Comment:
On Figma, the warning callout has an icon as emoji. Not sure if it's a good idea. Maybe we should use svg here instead? 

![image](https://user-images.githubusercontent.com/11715931/167485420-d59b6216-ee27-4be8-9682-611af85ceb9a.png)

Reply:
What's the safer option?

Reply:
as discussed - ask our UX specialists, but leave emoji for now